genai-microservices-connector/.golangci.yml

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
@chensuyue to check the license header. Need to align across OPEA projects

Reply:
Let's use this one.